### DISCUSSION TITLE
Overview - EKG/MM

### DISCUSSION THREAD

**Discussion by giscus** [EKGF/ekg-maturity#57] [General]
# Overview - EKG/MM

Maturity Model for the Enterprise Knowledge Graph (EKG)

https://maturity-model.ekgf.org/intro/overview/

**Comment by Alan-Michaels** [comment]
A.2.1 Market Segmentation should be renamed to Industry Segmentation which includes both market (buyer) segmentation + product segmentation (which you have included as measurable).
